src/storage/btree_load.c

```diff
@@ -3878,3 +3750,457 @@ btree_get_perf_btree_page_type (THREAD_ENTRY * thread_p, PAGE_PTR page_ptr)
     }
   return PERF_PAGE_BTREE_ROOT;
 }
+
+/*
+ * btree_load_check_fk () -
```

Comment:
Update comment & signature:

1. add short function description
2. remove _local suffix from load_args and sort_args.
3. loag_args: context for loaded index (includes leaf level of foreign key)
    sort_args: context for sorting (includes info on primary key)